src/Domain/Details.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Domain;
+
+final class Details
+{
+    /** @var mixed */
+    private $original;
+
+    /** @var string */
+    private $file;
+
+    /** @var int */
+    private $line;
+
+    /** @var string  */
+    private $function;
+
+    /** @var string */
+    private $message;
+
+    public static function make(): Details
+    {
+        return new Details();
+    }
+
+    /**
+     * @param string $file
+     *
+     * @return Details
```

Comment:
Detail: I'm wondering if this docblock is really necessary ?  We have all theses information in `public function withFile(string $file): Details`

Reply:
Totally true, I'll delete them! :)